### DISCUSSION TITLE
Emphasis styling on a navbar menu

### DISCUSSION THREAD

**Discussion by khalyomede** [twbs/bootstrap#39416] [Q&A]
[navbar documentation](https://getbootstrap.com/docs/5.2/components/navbar/)

I have a navbar implemented in production on [Predzo](https://predzo.com). The navbar works very well. 

![Current navbar](https://github.com/twbs/bootstrap/assets/15908747/e2282cfe-6b5c-43eb-86c0-998bfe0ccecc)

As the navbar starts to have a few menus, I would like to drive the focus of the user on the "Plans" menu. 

Maybe we could have a new class to emphasis a menu :

```html
<nav class="navbar navbar-expand-lg bg-light">
  <div class="container-fluid">
    <a class="navbar-brand" href="#">Navbar</a>
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
      aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
        <li class="nav-item">
          <a class="nav-link emphasized" aria-current="page" href="#">Home</a> <!-- here -->
        </li>
      </ul>
    </div>
  </div>
</nav>
```

I am no designer so I am not sure of the best practices, feel free to close this if it does not lead to any good accessible and/or good UI solutions.

I just would like to draw more attention on this specific menu to help driving more conversion to a paid plan (hopefully :crossed_fingers:).

I thought of putting a button, but there is already another button beside the brand name, and I find a button for a menu not suitable since this is a page (and the link can get an "active" class).

**Comment by julien-deramond** [comment]
I've converted your issue into a Q&A discussion for now @khalyomede. Based on the answers of the community and what could be found to address this topic, we might create an issue afterward if something not existing in Bootstrap is specified and can be generic enough to be added to the framework.

I am no designer either but here are some quick thoughts:
- you could move the "Plans" as a separate link in the center of the navbar in desktop mode to emphasize it is "different"/more important, or separate it from the rest of the header items with a vertical rule, something like: "Store News Admin | Plans"
- on top of that, there's also a possibility to use different font weights or font colors depending on what the nav links rendering is when active, and also if your color palette allows you to make it contrasted enough for accessibility

**Reply by khalyomede** [reply]
Thank you Julien, and your suggestions are very interesting, I need to brainstorm on this a little bit and give this another try 👍
